.github/workflows/update-production.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build and Update on production
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+    secrets: inherit
+
+  dispatch:
+    name: Dispatch to acm-aws
+    needs: build
+    runs-on: ubuntu-latest
+    environment: Production
+    concurrency: Production
+    steps:
+      - name: Dispatch workflow
+        uses: diamondburned/acm-aws/.github/actions/update-pkg@ebd2958df9430fbbf45a00efb1d2287db154f34e
```

Comment:
Use `@main` instead.

Reply:
[Done](https://github.com/acmcsufoss/sendlimiter/commit/33bd109ef0859a76f5f80ef8537d8569dc39ed9e)! Thanks for the review.